static/app/components/smartSearchBar/index.tsx

```diff
@@ -308,7 +309,7 @@ class SmartSearchBar extends React.Component<Props, State> {
     query: null,
     onSearch: function () {},
     excludeEnvironment: false,
-    placeholder: t('Search for events, users, tags, and everything else.'),
+    placeholder: t('Search for events, users, tags, and more'),
```

Comment:
The font has changed to a monospace one, I've shorted this up since some places had a input. that was just a bit too small for this